LogMonitor/LogMonitorTests/ConfigFileParserTests.cpp

```diff
@@ -65,6 +65,37 @@ namespace LogMonitorTests
             return str;
         }
 
+        ///
+        /// Add the path of the created directories, to be removed during
+        /// cleanup.
+        ///
+        std::vector<std::wstring> directoriesToDeleteAtCleanup;
+
+       ///
+       /// Creates a new random-name directory inside the Temp directory. 
+       ///
+       /// \return The new directory path. If an error occurs, it's empty.
+       ///
+        std::wstring CreateTempDirectory()

```

Comment:
Any chance we can make the function in LogFileMonitorTests public and use that or create a test utilities  file and add functions we reuse across tests there?